storage/memory/memory.go

```diff
@@ -274,6 +275,63 @@ func (c *checker) CheckAndUpdate(p *predicate.Predicate) bool {
 	return true
 }
 
+// latestFilter executes the latest filter operation over memoryTriples following filterOptions.
+func latestFilter(memoryTriples map[string]*triple.Triple, pQuery *predicate.Predicate, filterOptions *filter.StorageOptions) (map[string]*triple.Triple, error) {
```

Comment:
Looks like 'map[string]*triple.Triple' is used in multiple places. Maybe create a type alias as well, wdyt?

Reply:
Thank you for the comment, Jose. I thought about this too, but given the way the implementation of this volatile driver in `memory.go` was thought and how this `map[string]*triple.Triple` is used in some places with not exactly the same meaning, I believe it may be clearer to just let it transparent instead of abstracting it through a type alias. But lmk through chat if you still have any other comments.